WordPress/src/main/java/org/wordpress/android/ui/prefs/RecyclerViewItemClickListener.java

```diff
@@ -0,0 +1,61 @@
+package org.wordpress.android.ui.prefs;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.GestureDetector;
+import android.view.MotionEvent;
+import android.view.View;
+
+public class RecyclerViewItemClickListener implements RecyclerView.OnItemTouchListener {
+    private OnItemClickListener mListener;
+
+    public interface OnItemClickListener {
+        public void onItemClick(View view, int position);
```

Comment:
The methods of interface are public by definition, so we could safely remove the public modifier.